package.json

```diff
@@ -49,7 +49,7 @@
     "classnames": "^2.1.5",
     "debug": "^2.2.0",
     "lodash": "^4.6.1",
-    "react-portal": "^2.2.1",
+    "react-dom": "^15.3.1",
```

Comment:
Hm, thinking it through I don't believe we can get away without it.  Though, we can at least remove it from deps and just import it.  It is listed in `peerDependencies` with a range of supported versions.  This way, users can bring their own react / react-dom and we just import it, rather than bundling it (deps).

I'm not versed on the unstable API, but I wonder why the original portal didn't use `ReactDOM.render()` and just treat it as it's own app.  I should probably watch his talk referenced in the README, https://www.youtube.com/watch?v=z5e7kWSHWTg. :/